test/mksymbols.rb

```diff
@@ -42,4 +42,3 @@
 #{o}
 ];
 FOO
-
```

Comment:
github atom editor doesn't let me fix this file. that's what i've been using lately.

i think there is a line when you pull it down. whatever i use vim to fix that